config/boot.rb

```diff
@@ -24,9 +24,12 @@
 
 begin
   require 'bundler/setup'
-rescue LoadError
+rescue LoadError => e
   $stderr.puts "[*] Metasploit requires the Bundler gem to be installed"
   $stderr.puts "    $ gem install bundler"
+  $stderr.puts " You may need to install a specific version of bundler as well:"
+  $stderr.puts
+  $stderr.puts e
```

Comment:
Instead of the rather misleading message we get from bundler here, maybe we should guard a more specific error message (like downgrade to 1.17.2 or upgrade to 2.0.1+) with a `if RUBY_VERSION.start_with?('2.6')`.

Reply:
> with a if RUBY_VERSION.start_with?('2.6').

Was the issue specific to Ruby 2.6.x ?

Reply:
More specifically, the combination of rubygems 3 (ships by default in 2.6) and bundler 1.17.3. The other bundler 1.17.x and >=2.0.1 releases work fine. We should probably check `Gem::VERSION`, now that I think about it (edited above to be better).

Reply:
https://www.youtube.com/watch?v=lOfZLb33uCg

Reply:
Addressed, updating description.

Reply:
I didn't quite follow the suggestion of doing Ruby/Gem-version specific checks, but I think this should work for current and future snafus.